package.json

```diff
@@ -11,12 +11,18 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@looker/components": "^0.11.0",
-    "@looker/components-test-utils": "^0.9.17",
-    "@looker/design-tokens": "^0.7.2",
+    "@looker/components": "^1.1.3",
+    "@looker/components-test-utils": "^1.1.3",
+    "@looker/design-tokens": "^1.1.3",
     "@looker/extension-sdk": "^0.11.0",
     "@looker/extension-sdk-react": "^0.6.2",
+    "@looker/icons": "1.1.3",
     "@looker/sdk": "^0.3.0-beta.1",
+    "@styled-icons/boxicons-regular": "^10.23.0",
```

Comment:
Get rid of non material and looker icons?

Reply:
Oops. I always neglect the package.json